book.js

```diff
@@ -0,0 +1,45 @@
+// import "tailwindcss/tailwind.css"
+const myLibrary = [];
+
+function Book(author, title, pageCount, hasRead) {
+  const book = {
+    author,
+    title,
+    pageCount,
+    hasRead,
+  };
+  return book;
+}
+
+function addBookToLibrary() {
```

Comment:
- [ ] [OPTIONAL] Add validations for the input fields especially the pages input